integration-test/src/test/helpers/execAutifyCli.ts

```diff
@@ -21,7 +21,8 @@ const normalize = (stdout: string) =>
       /debug server on http:\/\/127.0.0.1:\d+/g,
       "debug server on http://127.0.0.1:<random>"
     )
-    .replace(/Your session ID is "[^"]+"/, 'Your session ID is "fake"');
+    .replace(/Your session ID is "[^"]+"/, 'Your session ID is "fake"')
+    .replace(/»/g, "›");
```

Comment:
oclif uses `»` on Win instead of `›`

https://github.com/oclif/core/blob/a18803454449986e03549838ba3928c66f98da37/src/errors/errors/cli.ts#L71